### PR TITLE
Includes a basic approach to the new icon radio group without icons a…

### PR DESCRIPTION
**Pull request by cumbreras**
…nd small version

This is a new version to use it on the new binder requoting feature:
 Alt text

Alt text

### CODE REVIEW

File: lib/components/IconRadioInput.js [hunk 1]
```diff
@@ -40,6 +41,7 @@ class IconRadioInput extends React.Component {
       'icons-radio-input',
       { 'icons-radio-input--checked': this.state.checked },
       { 'icons-radio-input--no-touch': !isIOS() },
+      { 'icons-radio-input--small': this.state.size === 'small' },
```
Comment (Quimera996) [inline lib/components/IconRadioInput.js]: small eh :trollface:

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+### References
+
+* **Pivotal Story:** _link related pivotal story_
```
Comment (dzmitry-kremez-itechart) [inline .github/PULL_REQUEST_TEMPLATE.md]: Opensource lib. Pivotal story?

File: lib/components/IconRadioInput.js [hunk 3]
```diff
@@ -7,6 +7,7 @@ class IconRadioInput extends React.Component {
     super(props);
     this.state = {
       checked: this.checked(props.value, props.checkedValue),
+      size: props.size,
```
Comment (dzmitry-kremez-itechart) [inline lib/components/IconRadioInput.js]: no need to put size in state, just use `this.props.size`, as it won't change over time of component lifecircle

File: lib/components/IconRadioInput.js (hunk 1 above)
Comment (dzmitry-kremez-itechart) [inline lib/components/IconRadioInput.js]: no need to put size in state, just use `this.props.size`

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (cumbreras) [inline .github/PULL_REQUEST_TEMPLATE.md]: I've got a ticket for this :D

**Pull request merged**